src/xrGame/saved_game_wrapper.cpp

```diff
@@ -74,6 +74,7 @@ CSavedGameWrapper::CSavedGameWrapper(LPCSTR saved_game_name)
         m_actor_health = 1.f;
         m_level_id = _LEVEL_ID(-1);
         m_level_name = "";
+        m_valid_save = false;
```

Comment:
I think it would be better to initialize 'default-invalid' values directly in the header (or in the inilialization list). It should help us to keep the state of the object consistent and to prevent some stupid errors in the future.

Reply:
I think this is should be next PR, like code-style.